wallet/service/openapi.json

```diff
@@ -0,0 +1,95 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Wallet API",
+    "description": "The version 2 of the wallet HTTP API leverages the wallet JSON-RPC API.\n\nSome useful links:\n- [Documentation of the wallet JSON-RPC API](https://docs.vega.xyz/docs/mainnet/api/jsonrpc/wallet)\n- [JSON-RPC 2.0 Specification](https://www.jsonrpc.org/specification)",
```

Comment:
**question (non-blocking)**: @candida-d I think this URL works, what do you think?

Reply:
Generally yes - though with the 'mainnet' path it won't be active until this gets released to mainnet, so it might be a while. The alternative is make it testnet and then update this link once the validators release all this to mainnet, or have 2 versions in the useful links, one for whatever version is on testnet and one for whatever version is on mainnet.

Reply:
so what should I do ?

Reply:
Which of the options I suggested do you think is most feasible? Having 2 links now, one which won't work for a while, or changing it later?

Reply:
I'm not sure which is best btw, I think neither is an ideal solution.

Reply:
I prefer to remove it if we can't make it work now. Broken link is no better than no link.